notebook/gen_ai_jss.ipynb

```diff
@@ -0,0 +1,670 @@
+{
```

Comment:
<p>Good thing I still pass this through to the second call! https://github.com/GoogleCloudPlatform/terraform-genai-doc-summarization/blob/c86b3d6505ac06fe1b0bfb99d91b7593f3caa39f/webhook/main.py#L115</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-genai-doc-summarization/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>